<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecWindow</class>
 <widget class="QMainWindow" name="SpecWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Bank_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Предпріятіе:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddTransfer">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>590</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Оформить ​заявку на переводъ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>590</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выходъ</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="AccTable">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>411</y>
      <width>541</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>641</width>
      <height>641</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Salary_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Заявки на ​зарплатный​ проектъ: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-120</x>
      <y>20</y>
      <width>471</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Images.qrc">:/LoginImage/resource/images/Sena-Saber-6.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Acc_lable">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Счета предпріятія: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CompanyName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AproveSalary">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Подать документы</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="SalaryTable">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>151</y>
      <width>541</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="AproveIndex">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>571</width>
      <height>1051</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Images.qrc">:/LoginImage/resource/images/Matou_sakura.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>Background</zorder>
   <zorder>Bank_label</zorder>
   <zorder>AddTransfer</zorder>
   <zorder>Exit</zorder>
   <zorder>AccTable</zorder>
   <zorder>Salary_label</zorder>
   <zorder>Acc_lable</zorder>
   <zorder>CompanyName</zorder>
   <zorder>AproveSalary</zorder>
   <zorder>SalaryTable</zorder>
   <zorder>AproveIndex</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Images.qrc"/>
 </resources>
 <connections/>
</ui>
